TreeMaker/python/makeJetVars.py

```diff
@@ -325,8 +279,6 @@ def makeJetVarsAK8(self, process, JetTag, suff, storeProperties, SkipTag=cms.VIn
         setattr(process,"QGTagger"+suff,QGTagger)
         ak8floats.extend([
             'BasicSubstructure'+suff+':girth',
-            'BasicSubstructure'+suff+':momenthalf',
-            'BasicSubstructure'+suff+':ptdrlog',
             'NjettinessBeta1'+suff+':tau1etaAxis1',

```

Comment:
remove these four lines too, that should fix the exception

Reply:
Now I get
```
----- Begin Fatal Exception 27-Feb-2020 10:22:51 CST-----------------------
An exception of category 'UnknownUserFloat' occurred while
   [0] Processing  Event run: 1 lumi: 40 event: 30762 stream: 0
   [1] Running path 'WriteTree'
   [2] Prefetching for module TreeMaker/'TreeMaker2'
   [3] Calling method for module JetProperties/'JetPropertiesAK8'
Exception Message:
Requested UserFloat BasicSubstructureAK8:girth is not available! Possible UserFloats are:
NjettinessAK8Puppi:tau1 NjettinessAK8Puppi:tau2 NjettinessAK8Puppi:tau3 NjettinessAK8Puppi:tau4 QGTaggerAK8:axis1 QGTaggerAK8:axis2 QGTaggerAK8:ptD ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau4 ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass ak8PFJetsCHSValueMap:ak8PFJetsCHSSoftDropMass ak8PFJetsCHSValueMap:eta ak8PFJetsCHSValueMap:mass ak8PFJetsCHSValueMap:phi ak8PFJetsCHSValueMap:pt ak8PFJetsPuppiSoftDropMass ak8PFJetsPuppiSoftDropValueMap:nb1AK8PuppiSoftDropN2 ak8PFJetsPuppiSoftDropValueMap:nb1AK8PuppiSoftDropN3 ak8PFJetsPuppiSoftDropValueMap:nb2AK8PuppiSoftDropN2 ak8PFJetsPuppiSoftDropValueMap:nb2AK8PuppiSoftDropN3 jerFactor nb1AK8PuppiSoftDrop:ecfN2 nb1AK8PuppiSoftDrop:ecfN3 nb2AK8PuppiSoftDrop:ecfN2 nb2AK8PuppiSoftDrop:ecfN3 puppiSpecificAK8:HFEMPuppiMultiplicity puppiSpecificAK8:HFHadronPuppiMultiplicity puppiSpecificAK8:neutralHadronPuppiMultiplicity puppiSpecificAK8:neutralPuppiMultiplicity puppiSpecificAK8:photonPuppiMultiplicity puppiSpecificAK8:puppiMultiplicity updatedPatJetsTransientCorrectedAK8UpdatedJECAuxiliaryJECup updatedPatJetsTransientCorrectedAK8UpdatedJECAuxiliaryJERdown updatedPatJetsTransientCorrectedAK8UpdatedJECAuxiliaryJERup
----- End Fatal Exception -------------------------------------------------
```